src/Evolution/Systems/RadiationTransport/M1Grey/M1Closure.hpp

```diff
@@ -0,0 +1,86 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines functions to compute the M1 closure
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"  // IWYU pragma: keep
+
+// IWYU pragma: no_forward_declare Tensor
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace RadiationTransport {
+namespace M1Grey {
+
+/*!
+ * Compute the 2nd moment of the neutrino distribution function
+ * in the inertial frame (pressure tensor) from the 0th (energy density)
+ * and 1st (momentum density) moments. The M1 closure sets
+ * \f$P_{ij} = d_{\rm thin} P_{ij,\rm thin} + d_{\rm thick} P_{ij,\rm thick}\f$
+ * with \f$P_{ij}\f$ the pressure tensor in the inertial frame, and
+ * \f$P_{ij,thin/thick}\f$ its value assuming an optically thick / optically
+ * thin medium.
+ *
+ * Following the algorithm describe in appendix A of arXiv:1502.04146
+ * (Foucart et al 2015, PRD 91 124021), we choose the Minerbo closure
+ * which sets
+ * \f$d_{\rm thin}=1.5\chi-0.5\f$, \f$d_{\rm thick}=1.5-1.5\chi\f$
+ * with \f$\chi = \frac{1}{3} + \xi^2 \frac{6-2\xi+6\xi^2}{15}\f$
+ * and \f$\xi=H^aH_a/J^2\f$,  with \f$J\f$ the energy density in the
```

Comment:
Is xi different from zeta? The equations I'm looking at in the paper seem to use zeta as the variable for the closure function, we should try to be consistent unless there is a reason for the deviation.

Reply:
It was xi in the SpEC code but zeta in my paper... i changed it to zeta everywhere.